### PR TITLE
Docker image building

### PR DESCRIPTION
**Pull request by himadrisingh**

### CODE REVIEW

File: .github/workflows/cli-release.yml [hunk 1]
```diff
@@ -59,7 +61,7 @@ jobs:
               -v `pwd`/rill-binary-sa.json:/rill-binary-sa.json \
               -w /go/src/rilldata/rill-developer \
               goreleaser/goreleaser-cross:v1.19.2 \
-              release ${NIGHTLY}
+              release ${NIGHTLY} --snapshot
```
Comment (himadrisingh) [inline .github/workflows/cli-release.yml:64]:
```suggestion
              release ${NIGHTLY}
```

**Review by begelundmuller — CHANGES_REQUESTED** [review]
- I think we can get rid of `entrypoint.sh` file. Doesn't seem like something we use/do anymore.
- Let's copy the `rill` binary to `/rill`, not `/app/rill` (or maybe to a `bin` folder?)
- For the entrypoints, how about this? (Always forget the Docker mechanics for these, let me know if it looks bad):

```
ENTRYPOINT /rill
CMD /rill start
```
- Should we let the default workdir be `/project`? So people should mount their artifacts there if they have an existing project. (Or do you have a more idiomatic suggestion, maybe `/data`?) (People can always override by passing `--project /path/to/code` anyway)

File: .goreleaser.yaml [hunk 2]
```diff
@@ -90,6 +94,13 @@ brews:
       name: rilldata
       email: support@rilldata.com
 
+dockers:
+  - image_templates:
+      - rilldata/rill:latest
+      - rilldata/rill:{{ .Tag }}
+    extra_files:
+      - scripts/entrypoint.sh
```
Comment (himadrisingh) [inline .goreleaser.yaml:102]:
```suggestion
```

File: Dockerfile [hunk 3]
```diff
@@ -1,36 +1,9 @@
 # syntax = docker/dockerfile:1.1-experimental
 FROM ubuntu:focal
+WORKDIR /project
```
Comment (kaspersjo) [inline Dockerfile:4]: We need to `apt-get install -y ca-certificates` for the bucket and HTTP source connections to work.

**Review by kaspersjo — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -1,36 +1,9 @@
...
```
Comment (himadrisingh) [inline Dockerfile:3]:
```suggestion
RUN apt-get install -y ca-certificates
WORKDIR /project
```

**Review by begelundmuller — APPROVED** [review]
LGTM

**Pull request merged**